domains/auth/Auth.API/API/Program.cs

```diff
@@ -99,12 +101,33 @@
 builder.Services.AddScoped<IUserRepository, UserRepository>();
 builder.Services.AddScoped<IUserDataContext, DataContext>();
 
+builder.Services.AddOpenTelemetry()
+    .WithMetrics(provider =>
+        provider
+            .AddHttpClientInstrumentation()
+            .AddAspNetCoreInstrumentation()
+            .AddRuntimeInstrumentation()
+            .AddPrometheusExporter())
+    .WithTracing(provider =>
+        provider
+            .AddHttpClientInstrumentation()
+            .AddAspNetCoreInstrumentation()
+            .AddJaegerExporter(options =>
+            {
+                var config = builder.Configuration.GetSection(JaegerOptions.Prefix).Get<JaegerOptions>();
```

Comment:
Let us try to check if this section exist and only use it if it does. (... and then remove the one configured above)